### ISSUE TITLE
Expand/suppress renewal output based on preconfigured renewal flag

### ISSUE THREAD

**Issue by ohemorange** [certbot/certbot#8327]
`--preconfigured-renewal` was added in https://github.com/certbot/certbot/pull/8274. This flag is set in environments where Certbot should have automated renewal preconfigured. When the flag is set, we shouldn't need to print the following:

```
To obtain a new or tweaked
version of this certificate in the future, simply run certbot again
with the "certonly" option. To non-interactively renew *all* of
your certificates, run "certbot renew"
```

**Comment by ohemorange** [comment]
We should also set this in the snap probably

**Comment by ohemorange** [comment]
Closed by #8852.
